cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -941,6 +1036,55 @@ instance Show remotePeer => ToObject (TraceKeepAliveClient remotePeer) where
       dTime (Time d) = realToFrac d
 
 
+instance ToObject TraceLedgerPeers where
+  toObject _verb (PickedPeer addr _ackStake stake) =
+    mkObject
+      [ "kind" .= String "PickedPeer"
+      , "address" .= show addr
+      , "relativeStake" .= (realToFrac (unPoolStake stake) :: Double)
+      ]
+  toObject _verb (PickedPeers (NumberOfPeers n) addrs) =
+    mkObject
+      [ "kind" .= String "PickedPeers"
+      , "desiredCount" .= n
+      , "count" .= length addrs
+      , "addresses" .= show addrs
+      ]
+  toObject _verb (FetchingNewLedgerState cnt) =
+    mkObject
+      [ "kind" .= String "FetchingNewLedgerState"
+      , "numberOfPools" .= cnt
+      ]
+  toObject _verb DisabledLedgerPeers =
+    mkObject
+      [ "kind" .= String "DisabledLedgerPeers"
+      ]
+  toObject _verb (TraceUseLedgerAfter ula) =
```

Comment:
The p2p node can take root peers from the ledger.  Depending how it is configured it will either trace that this is disabled or enabled (after some slot).